release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -11,7 +11,7 @@ The release manager must:
   kubernetes/org to request membership
 * Be a top level approver for the repository. To become a top level approver,
   the candidate must demonstrate ownership and deep knowledge of the repository
-  through active maintainence, responding to and fixing issues, reviewing PRs,
+  through active maintenance, responding to and fixing issues, reviewing PRs,
```

Comment:
We can't directly modify changes under release-tools.  Please fix them in this repo: https://github.com/kubernetes-csi/csi-release-tools instead.  All changes under release-tools are imported from csi-release-tools repo like this: https://github.com/kubernetes-csi/external-snapshotter/pull/301